### DISCUSSION TITLE
Release 1.24

### DISCUSSION THREAD

**Discussion by webspiderteam** [webspiderteam/Bluetooth-DMM-For-Windows#38] [Announcements]
- Fixed Type 4 devices decoding
- Fixed Mode Buttons was not moving with main window
- Fixed wrong Mode buttons actions
- Added missing mode buttons
- Fixed Analog diplay stucks on last state when not numeric datas displayed
- Removed Red strip for device type 4 not tested nodification

<hr /><em>This discussion was created from the release <a href='https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/releases/tag/WPFRelease1.24'>Release 1.24</a>.</em>

**Comment by ludwich66** [comment]
Time for a Wiki Refresh ;-) next Days!
L

**Reply by webspiderteam** [reply]
Would be great 😊

**Comment by ludwich66** [comment]
Hello Engine,
where can I find the pictures of the remote buttons?
Can you please save them as SVG/JPG under Extras. (Complete ZIP)
Greetings Ludwich

**Reply by webspiderteam** [reply]
There is no image there some of them only text as stretched others are was SVG but there is only data's no SVG files but I can get a screenshot without measurements in background

**Reply by ludwich66** [reply]
Ok good idea, stretch it to max. I'm trying to convert them to svg again.
L

**Reply by webspiderteam** [reply]
![image](https://user-images.githubusercontent.com/85828505/213915267-74e5623f-b1d0-4902-97e2-bb2274d91938.png)
here it is

**Reply by webspiderteam** [reply]
![image](https://user-images.githubusercontent.com/85828505/213915589-86d947fe-75e0-408e-855b-294a77c0cc47.png)

**Reply by ludwich66** [reply]
There are differents?

**Reply by webspiderteam** [reply]
Only difference is added some space between two lines. Maybe you would need separate them

**Reply by webspiderteam** [reply]
there is more hidden data in 5566 series but I couldn't findout maybe from little display or maybe from analog display or it can be another measurement. BTW official app sends data without doing anything. maybe it is some kind of ping but its changing everytime.
